lib/ddr/vocab/asset.rb

```diff
@@ -1,29 +1,35 @@
- module Ddr
-  module Vocab
-    class Asset < RDF::StrictVocabulary("http://repository.lib.duke.edu/vocab/asset/")
+module Ddr::Vocab
```

Comment:
Other than addition of ingestedBy and ingestionDate terms, changes are formatting only.